packages/forms/src/styled/index.js

```diff
@@ -43,3 +43,11 @@ export { default as StyledToggleLabel } from './toggle/StyledToggleLabel';
 export { default as StyledToggleHint } from './toggle/StyledToggleHint';
 export { default as StyledToggleInput } from './toggle/StyledToggleInput';
 export { default as StyledToggleMessage } from './toggle/StyledToggleMessage';
+
+/**
+ * Range styles
+ */
+export { default as StyledRangeSingleThumb } from './range/StyledRangeSingleThumb';
+export { default as StyledRangeLabel } from './range/StyledRangeLabel';
+export { default as StyledRangeHint } from './range/StyledRangeHint';
+export { default as StyledRangeMessage } from './range/StyledRangeMessage';
```

Comment:
I think the problem with spacing is that these styled Label/Hint/Message components are not being used. Once these are configured, with corresponding context, in `Label.js`, `Hint.js`, and `Message.js` I think we'll see the spacing mentioned under my previous comments.